modules/nodes-edge-remote-workers-strategies.adoc

```diff
@@ -133,7 +131,12 @@ A taint with the `NoExecute` effect affects pods that are running on the node in
 
 * Pods that do not tolerate the taint are queued for eviction.
 * Pods that tolerate the taint without specifying a `tolerationSeconds` value in their toleration specification remain bound forever.
-* Pods that tolerate the taint with a specified `tolerationSeconds` value remain bound for the specified amount of time.  After the time elapses, the pods are queued for eviction.
+* Pods that tolerate the taint with a specified `tolerationSeconds` value remain bound for the specified amount of time. After the time elapses, the pods are queued for eviction.
+
+[NOTE]
+====
+Unless tolerations are explicitly set, Kubernetes will automatically add a toleration for `node.kubernetes.io/not-ready` and `node.kubernetes.io/unreachable` with `tolerationSeconds=300`, meaning that pods will remain bound for 5 minutes if either of these taints is detected. 
```

Comment:
```suggestion
Unless tolerations are explicitly set, Kubernetes automatically adds a toleration for `node.kubernetes.io/not-ready` and `node.kubernetes.io/unreachable` with `tolerationSeconds=300`, meaning that pods remain bound for 5 minutes if either of these taints is detected. 
```

Reply:
Avoid using future tense:
https://www.ibm.com/docs/en/ibm-style?topic=grammar-verbs#tense

Reply:
Fixed, thanks for the catch!